data-exporter/runDataExporter.sh

```diff
@@ -11,10 +38,17 @@ git pull
 config_file=config.properties
 ## If the config file does not exist, run configuration script
 if [ ! -f $config_file ]; then
-    ./configureDataExporter.sh
+	./configureDataExporter.sh
+fi
+
+jar_file="data-exporter.jar"
+## Generate the jar file if it doesn't exist or a re-build is requested
+if [ ! -f $jar_file ] || [ ! -z $build_jar ]; then
+	mvn clean package
```

Comment:
Is there any need for a git pull?

Reply:
The git pull happens earlier in the script.  I was going to put it in this block, but it should happen regardless of a build happening or not so that the configuration shell script is up to date.  Also, so the source code is up to date in case anything is done manually.